### DISCUSSION TITLE
Show me your Meow!

### DISCUSSION THREAD

**Discussion by trev-dev** [meow-edit/meow#283] [General]
I keep bouncing back-and-fourth between Meow and God-Mode + Multiple Cursors. Part of this is my ADHD pulling me all over the place. The other is maybe laziness. I am sure I could probably replicate many of the aspects of my neat god-mode config in Meow if I wanted, such as "Emacs Native" bindings for movement. 

I don't enjoy switching back and fourth between HJKL and FBNP for navigation. Some packages are opinionated enough that Meow doesn't "meld" as well as god-mode does.

I mostly have a default configuration at this point and would love some inspiration, especially on the QWERTY side of things. Sho wme what you've got!

**Comment by DogLooksGood** [comment]
I don't think the position of navigation keys is a barrier to choosing modal editing packages. Both Meow and God-mode allow you to customize the bindings. In my opinion, the issue here is whether to use a set of commands that are designed for modal editing. 
Meow provides
- a series of commands around the concept "selection", primarily for navigation and text manipulation. 
- a way to execute built-in bindings
- a feature to improve keyboard macros

The key bindings are not important at all, and yes, it's super difficult to have a perfect command layout.

**Reply by trev-dev** [reply]
Just want to highlight for other readers that it also provides:

* An intuitive macro abstraction that requires fewer commands/keystrokes than other packages with `beacon-mode`
* A "command mode" called `keypad-mode` for lowering key-chords into single presses. It does more for you out of the box than god can by inferring single-key presses after prefixes.
